docs/node-developers/codebase-overview.mdx

```diff
@@ -41,17 +41,17 @@ One difficult issue with learning OCaml is locating and reading documentation fo
     Unix  <-  Core  ->       Async
 ```
 
-There are actually multiple sources for `Core` documentation. To find the correct documentation, first go the GitHub then use the codebase to locate the correct documentation. If you can't find the module you're looking for go to `Core_kernel` and if that fails then look for `Base`. The find capability does work if the sections are not expanded.
+There are multiple sources for OCaml Core library documentation. In general, the source code of an installed library is not available so to find the  documentation, first go to the codebase on GitHub and then locate the correct documentation. If you don't see the module you're looking for, go to `Core_kernel`. If that fails, then look for `Base`. To use the find capability in GitHub, expand the sections.
 
-Instead of using the HTML documentation, a better way is to use editor integration using a tool like `merlin` that provides type hints. One downside though is that Merlin works only if your code compiles. It also gives us code navigation like go to definition.
+The [Core library documentation](https://v2.ocaml.org/manual/core.html) is published in HTML. However, you can use the Merlin editor service that provides modern IDE features for OCaml. Merlin provides type hints and code navigation, like Go To Definition. Note that Merlin works only if your code compiles. 
```

Comment:
You could DM Deepthi to clarify. She leads protocol eng team

Reply:
@deepthiskumar I'll ping you on Slack to introduce myself and to ask for your help in making sure the doc talks about the right "core" and what library docs we are guiding our users to. 
Let's sync up!